src/Modal/Modal.js

```diff
@@ -118,6 +121,7 @@ function Modal({ children, title, animationDuration, showClose, onClose, open, .
 }
 
 Modal.propTypes = {
+  groupId: PropTypes.string.isRequired,
```

Comment:
What is groupId's purpose and could we get away without it? This would be a somewhat larger change given that we'd have to go and add groupId's to every existing modal.

Reply:
Thanks for pointing that out, this is zombie code from testing [a recommended approach](https://github.com/theKashey/react-focus-lock#portals) for handling portalled content, and has been removed.